sysml.library/Kernel Library/Clocks.kerml

```diff
@@ -44,7 +44,11 @@ package Clocks {
 	 * which is considered to be synchronized with the snapshot of the Clock 
 	 * with a currentTime equal to the returned timeValue.
 	 */
-	abstract function TimeOf(o : Occurrence[1], clock : Clock[1]) timeValue : ScalarValue[1] {
+	abstract function TimeOf {
+		in o : Occurrence[1];
+		in clock : Clock[1] default defaultClock;
+		return timeValue : ScalarValue[1];
+		
```

Comment:
I would prefer using `timeInstantValue` rather than just `timeValue` to stress the difference with `duration`. Even in ISO 80000-3 `duration` is aliased to `time`, and many people do this. Humans mostly intuitively understand whether "time something takes" vs "the time at which something happens" is intended, but in the library I think we should be rigorous.